osf/models/preprint_service.py

```diff
@@ -122,7 +144,7 @@ def get_subjects(self):
         return ret
 
     def set_subjects(self, preprint_subjects, auth):
-        if not self.node.has_permission(auth.user, ADMIN):
+        if not self.has_permission(auth.user, 'ADMIN'):
```

Comment:
Not sure checking for the hard coded string `'ADMIN'` is the way to go here (and the other places) --  is there  a way to check for permissions still using a configurable constant? After all that imported `ADMIN` constant is in the end 'admin', but it's set one place